tables/ru-ru-g1.ctb

```diff
@@ -0,0 +1,664 @@
+# liblouis: Russian literary (grade 1) text translation table
+#
+#-index-name: Russian, grade 1
+#-display-name: Russian contracted braille
+#
+#+locale: ru
+#+type: literary
+#+dots: 6
+#+contraction: partial
+#+grade:1
+#+direction: forward
+#
+#  Copyright (C) 2021 Kate <musicate@yandex.ru>
+#  Copyright (C) 2021 Andrey Yakuboy <andrewia2002@yandex.ru>
+#
+#  This file is part of liblouis.
+#
+#  liblouis is free software: you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as
+#  published by the Free Software Foundation, either version 2.1 of the
+#  License, or (at your option) any later version.
+#
+#  liblouis is distributed in the hope that it will be useful, but
+#  WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with liblouis. If not, see
+#  <http://www.gnu.org/licenses/>.
+#
+#Based on Zubkov M. - "Contracted braille. Grade 1" (Moscow, 1974)
```

Comment:
Is this document available somewhere? Should we include, reference or mention it from https://github.com/liblouis/braille-specs/tree/master/russian?

Reply:
Also, is there some more background we could give here? Maybe something along the lines of what you said in the PR comment (that it is not in use officially and not many people know it)?

Reply:
> Is this document available somewhere?

Initially it's only braille book. Month ago I digitized it, but the electronic text hasn't been proofread. I think I will be able to pull these books (grade 1 and grade 2) later.

> Also, is there some more background we could give here?

Will do bit later.

Reply:
@Futyn-Maker Do you have time now to add that bit of background? We're trying to finish up the release now because we have no time in the weekend.